students/Deniss_Semcovs/Lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+data = [
+    ["Tom Hanks", 1000, 500],
+    ["Leonardo DiCaprio", 300, 1200, 500],
+    ["Natalie Portman", 5000],
+    ["Harrison Ford", 500, 2000, 100],
+    ["Scarlett Johansson", 1500, 1000],
+    ["Jennifer Aniston", 2000, 2000, 2000],
+]
```

Comment:
Need to use a dictionary for your user database in this version of mailroom.  Please revise and resubmit for another review.